packages/world/ts/config/v2/worldWithShorthands.ts

```diff
@@ -33,7 +34,9 @@ export type validateWorldWithShorthands<world> = {
   [key in keyof world]: key extends "tables"
     ? validateTablesWithShorthands<world[key], extendedScope<world>>
     : key extends "namespaces"
-      ? validateNamespacesWithShorthands<world[key], extendedScope<world>>
+      ? world extends { namespace?: unknown; tables?: unknown; systems?: unknown }
+        ? ErrorMessage<`\`Can only use \`namespaces\` with \`namespace\`, \`tables\`, or \`systems\` keys.`>
```

Comment:
```suggestion
        ? ErrorMessage<"Cannot use `namespaces` with `namespace`, `tables`, or `systems` keys.">
```
also wondering if there's a way to not duplicate this behavior here